helm/charts/determined/values.yaml

```diff
@@ -22,14 +22,26 @@ useNodePortForMaster: false
 # Enable route support for Openshift by setting enabled to true. Configure tls termination (i.e edge) if needed.
 # openshiftRoute:
   # enabled:
+  # host:
   # termination:
 
-# tlsSecret enables TLS encryption for all communication made to the Determined master (TLS
-# termination is performed in the Determined master). This includes communication between the
-# Determined master and the task containers it launches, but does not include communication between
-# the task containers (distributed training). The specified Secret of type tls must already exist in
-# the same namespace in which Determined is being installed.
-# tlsSecret:
+# security:
+  # tlsSecret enables TLS encryption for all communication made to the Determined master (TLS
+  # termination is performed in the Determined master). This includes communication between the
+  # Determined master and the task containers it launches, but does not include communication between
+  # the task containers (distributed training). The specified Secret of type tls must already exist in
+  # the same namespace in which Determined is being installed.
+  # tlsSecret:
+
+  # defaultTask sets the user and group that tasks will run as. For convenience, the default Determined
+  # environments contain an unprivileged user named det-nobody, which does have a writable HOME
+  # directory. The det-nobody user is a suitable default user when using the default Determined
+  # environment images and when running containers as root is not desired.
+  # defaultTask:
+  #   user: det-nobody
+  #   uid: 65533
+  #   group: det-nobody
+  #   gid: 65533
```

Comment:
I forget - is there a good reason to not use `-3` here so it'll overflow properly on both systems with 32-bit IDs and older systems with 16-bit IDs?  Is `65533` hard-coded elsewhere in the containers?

Reply:
65533 is hardcoded in our containers as `det-nobody`, I believe.